webapp/components/interop.js

```diff
@@ -172,10 +172,15 @@ class InteropDataManager {
           testScore += score;
         });
 
-        let summaryScore = testScore / numFocusAreas;
+        testScore /= numFocusAreas;
 
         // Handle investigation scoring if applicable.
-        summaryScore = this.#addInvestigationScore(summaryScore, date);
+        const [investigationScore, investigationWeight] =
+          this.#getInvestigationScoreAndWeight(date);
+
+        // Factor in the the investigation score and weight as specified.
+        let summaryScore = testScore * (1 - investigationWeight);
+        summaryScore += investigationWeight * investigationScore;
 
         summaryScore = Math.floor(summaryScore);
```

Comment:
This might be even simpler to understand as a single expression:

```suggestion
        const summaryScore = Math.floor(testScore * (1 - investigationWeight) +
                                        investigationScore * investigationWeight);
```